app/policies/subject_policy.rb

```diff
@@ -9,9 +9,7 @@ def manage_assessments?
 
   class Scope < Scope
     def resolve
-      Subject.
-        with_direct_assessments.
-        where(department_number: user.departments.pluck(:number))
+      Subject.where(department_number: user.departments.pluck(:number))
```

Comment:
It should not. I made a corresponding change adding this scope back in to where we need it - the subjects controller.